src/DataBase/DataBase.cc

```diff
@@ -1923,7 +1938,6 @@ globalSamplingBoundingVolume(typename Dimension::Vector& centroid,
 				    xminNodes, xmaxNodes,
 				    xminSample, xmaxSample);
 
-#ifdef USE_MPI
```

Comment:
@ptsuji @mdavis36 Should this ifdef be removed? It seems like it was here before and might be intentional.

Reply:
This one is intentional and should be left in.

Reply:
Ok, I've reverted this change.